src/FlagshipContext.tsx

```diff
@@ -139,15 +148,20 @@ export const FlagshipProvider: React.FC<FlagshipProviderProps> = ({
   initialBucketing,
   initialCampaigns,
   initialModifications,
+  initialFlags,
   synchronizeOnBucketingUpdated,
   activateDeduplicationTime,
   hitDeduplicationTime,
   visitorCacheImplementation,
   hitCacheImplementation,
   disableCache
 }: FlagshipProviderProps) => {
-  let modifications = new Map<string, Modification>()
-  if (initialModifications) {
+  let modifications = new Map<string, FlagDTO>()
+  if (initialFlags) {
+    initialFlags.forEach((flag) => {
```

Comment:
Will this foreach work even if the initialFlags props is of type Map<string, FlagDTO> (which it can be according to the type definition) ?